packages/cli-platform-ios/src/commands/buildIOS/buildProject.ts

```diff
@@ -14,12 +11,11 @@ import {
 export type BuildFlags = {
   mode: string;
   target: string;
-  packager: boolean;
   verbose: boolean;
   xcconfig?: string;
   buildFolder?: string;
   port: number;
```

Comment:
Doesn't port end up inside `RCT_METRO_PORT` at compile time for iOS (so may still be needed)? https://github.com/facebook/react-native/blob/080a3d300dc357fec62d767d140f964f9db66b2e/packages/react-native/React/Base/RCTDefines.h#L130

Reply:
It's still going to be set for `run-ios`. For `build-ios` we don't expect the Metro server to run because the JS bundle will be, well, bundled into the app executable

Reply:
Yup, it doesn't make sense to pass these parameters to `build-ios` command as it doesn't use it.